browser/src/control/Control.DownloadProgress.js

```diff
@@ -64,6 +64,9 @@ L.Control.DownloadProgress = L.Control.extend({
 				msg, buttonText, this._onStartDownload.bind(this), this.options.snackbarTimeout);
 
 			this.setupKeyboardShortcutForSnackbar();
+
+			// Snackbars cannot get focus, but we are using it with a button and it requires to be focused. No need to change snackbar imp for now.
+			document.getElementById('button').focus(); // TODO: This "button" id is too generic. It could be something like "snackbar-button" etc.
```

Comment:
https://github.com/CollaboraOnline/online/pull/10394/commits/213974e6619b55010008d3aba17f5277201d0f42 has a check here for the case when the getElementById() result is undefined, is it intentional that you don't do the same here? Maybe extract to a function to avoid the duplication?